tests/test_scripts/mf6_water_balance/MF6_wbal_listing.py

```diff
@@ -31,6 +31,7 @@ def listfile_to_dataframe(file_in: Path) -> pd.DataFrame:
                 postfix = "_IN"
             elif re.match(r"^\s*VOLUME.* BUDGET.*STRESS PERIOD\s+(\d+)", line):
                 m = re.match(r"^\s*VOLUME.* BUDGET.*STRESS PERIOD\s+(\d+)", line)
+                assert m is not None  # for mypy
```

Comment:
```suggestion
            elif m := re.match(r"^\s*VOLUME.* BUDGET.*STRESS PERIOD\s+(\d+)", line):
```

I think that should do the trick as well

Reply:
done